### DISCUSSION TITLE
How to support multiple users on Windows Server?

### DISCUSSION THREAD

**Discussion by ngugcx** [rustdesk/rustdesk#2441] [Q&A]
I have a remote server of Windows Server.
There are multiple users, I want to use rustdesk like rdp, multiple users can connect to their own desktop at the same time.
But it seems that rustdesk only supports one desk on a server, it allow only one user.
Correct me if I'm wrong.
Thanks.

**Comment by HyP3r-** [comment]
This is pretty similar to mine: https://github.com/rustdesk/rustdesk/discussions/2095
